recipes/openfx/all/conanfile.py

```diff
@@ -0,0 +1,91 @@
+from conans import ConanFile, CMake, tools
+
+import os
+import textwrap
+
+
+class openfx(ConanFile):
+    name = "openfx"
+    version = "1.4.0"
```

Comment:
```suggestion
```

Do not declare version in the recipe. It should be collected from conandata.yml, so the recipe can be re-used for any version.